source/extensions/retry/host/well_known_names.h

```diff
@@ -17,8 +17,20 @@ class RetryHostPredicatesNameValues {
   // Previous host predicate. Rejects hosts that have already been tried.
   const std::string PreviousHostsPredicate = "envoy.retry_host_predicates.previous_hosts";
   const std::string OmitCanaryHostsPredicate = "envoy.retry_host_predicates.omit_canary_hosts";
+  const std::string OmitHostsPredicate = "envoy.retry_host_predicates.omit_hosts";
 };
 
+/**
+ * Well-known metadata filter namespaces.
+ */
+class MetadataFilterValues {
+public:
+  // Filter namespace for built-in load balancer.
+  const std::string ENVOY_LB = "envoy.lb";
```

Comment:
i think this already exists in source/common/config/well_known_names.h, can you reuse that instead?

Reply:
My bad, I didn't revert the changes I was trying locally. Cleaned up and reused the existing code.